contracts/UniversalV3Staker.sol

```diff
@@ -107,19 +108,20 @@ contract UniversalV3Staker is IUniversalV3Staker, Multicall {
     /// @param _nonfungiblePositionManager the NFT position manager contract address
     /// @param _maxIncentiveStartLeadTime the max duration of an incentive in seconds
     /// @param _maxIncentiveDuration the max amount of seconds into the future the incentive startTime can be set
-    /// @param _cfNbits cumulative function tree parameter
     constructor(
         IUniswapV3Factory _factory,
         INonfungiblePositionManager _nonfungiblePositionManager,
         uint256 _maxIncentiveStartLeadTime,
-        uint256 _maxIncentiveDuration,
-        uint256 _cfNbits
+        uint256 _maxIncentiveDuration
     ) {
         factory = _factory;
         nonfungiblePositionManager = _nonfungiblePositionManager;
         maxIncentiveStartLeadTime = _maxIncentiveStartLeadTime;
         maxIncentiveDuration = _maxIncentiveDuration;
-        cfNbits = _cfNbits;
+
+        uint24 numTicks = uint24(TickMath.MAX_TICK - TickMath.MIN_TICK + 1);
```

Comment:
numTicks may need to add +2 since 0 should be also included?

Reply:
good call. done, also added comments for explanation.